react/Components/Footers/FooterDesktop.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react'
+import style from "./FooterDesktop.css"
+
+interface FooterProps {
+
+}
+
+const FooterDesktop: StorefrontFunctionComponent<FooterProps> = (props: any) => {
+    const footerSectionItems = [
+        {
+            title: "Shop",
+            items: props.shopLinks ? props.shopLinks : []
+        },
+        { title: "About", items: props.aboutLinks ? props.aboutLinks : [] },
+        { title: "Help", items: props.helpLinks ? props.helpLinks : [] }
+    ]
+    console.log(footerSectionItems);
+
+    return (<div>
+        <div className={`${style.title} ph7 pt4`}>
```

Comment:
Use vtex tachyons instead of writing the custom CSS and please refer to the doc [https://tachyons.io/docs/table-of-styles/](https://github.com/akshay-trika/training-task/pull/1/url)

Ex: `<div className="flex justify-between"> <div>`